guides/common/modules/proc_creating-a-virt-who-configuration.adoc

```diff
@@ -0,0 +1,131 @@
+** *Filter hosts*: Comma-separated list of included hypervisors.
+** *Exclude hosts*: Comma-separated list of excluded hypervisors.
++
+--
+Specify the host name or UUID according to the hypervisor ID you selected.
+
+.Host names
+
+* You can use wildcards, regular expressions, and special characters in the host name.
+* If you use regular expressions, you must escape the backslashes.
+* If you use special characters, you must enclose the host name in quotation marks.
+--
++
+* vCenter Server:
+** *Filter host parents*: Comma-separated list of included cluster IDs.
+** *Exclude host parents*: Comma-separated list of excluded cluster IDs.
+For more information, see link:https://access.redhat.com/solutions/5696481[Using the "Filter Host Parents" and "Exclude Host Parents" Attributes with VMware Clusters] in the _Red{nbsp}Hat Knowledgebase_.
+
+. You can configure the following logging and proxy options:
+
+* *Enable debugging output*: Enables debug logging for virt-who.
+* *HTTP Proxy*.
+Example: `http://_proxy.example.com_:3128`.
++
+To use no proxy, leave this field blank; this has the same result as entering `{asterisk}` in the *Ignore Proxy* field.
+* *Ignore Proxy*: Comma-separated list of host names, IP addresses, or domains to bypass existing proxy settings.
+. {KubeVirt}: Enter the path in the *Path to kubeconfig file* field.
+. Nutanix AHV:
+* Select *Prism Central* or *Prism Element* from the *Prism Flavor* list.
+* Optional: *Enable AHV Debug*: Enables AHV internal debugging.
+This option provides additional AHV information when you enable both debugging options.
+
+. Click *Submit*.
+
+.CLI procedure
+
+* On {ProjectServer}, enter the `hammer virt-who-config create` command according to the following example:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer virt-who-config create \
+--name _My_virt-who_Configuration_ \
+--organizations "_My_Organization_" \
+--interval 720 \ <1>
+--filtering-mode none \ <2>
+--hypervisor-id hostname \ <3>
+--hypervisor-type libvirt \ <4>
+--hypervisor-server _{foreman-example-com}_ \ <5>
+--hypervisor-username virt_who_user \ <6>
+--proxy '_http://proxy.example.com_:3128' \ <7>
+--satellite-url _server.example.com_
+----
+--
+<1> Virtual machine information reporting interval, in minutes.
+<2> Specify `none` for no filtering of hypervisors for virt-who queries.
+Specify `whitelist` or `blacklist` to include or exclude hypervisors for virt-who queries.
+<3> Specify `hostname`, `uuid`, or `hwuuid` for the hypervisor ID format.
+* You can use `uuid` to avoid duplication if a hypervisor is renamed.
++
+* You can use `hwuuid` for configurations that apply to a virtualization manager instead of an individual hypervisor.
++
+[NOTE]
+====
+You cannot change `hwuuid` to another option after virt-who starts running because this might cause duplicate entires in Subscription Manager.
+====
+
+<4> Specify the hypervisor type:
+* {EL} Virtualization (KVM), {oVirt}, or {OpenStack}: `libvirt`.
```

Comment:
Drop {oVirt}

Reply:
@chris1984 Is oVirt and RHV still supported by Foreman/Katello nightly (& Satellite latest/upcoming only if you can answer publicly)? 

My current understanding is "yes": https://docs.theforeman.org/nightly/Provisioning_Hosts/index-katello.html#Provisioning_Virtual_Machines_on_oVirt_rhv-provisioning & https://docs.theforeman.org/nightly/Provisioning_Hosts/index-satellite.html#Provisioning_Virtual_Machines_on_oVirt_rhv-provisioning